File: src/main/java/pl/weronika/kurczyna/model/entity/JobGrade.java
package pl.weronika.kurczyna.model.entity;

import org.springframework.lang.Nullable;

import javax.persistence.*;

@Entity(name = "jobGrades")
@Table(name = "JOB_GRADES")
public class JobGrade {
    @Id
    @Nullable
    @Column(unique = true, name = "GRADE_LEVEL")
    private String gradeLevel;
    @Nullable
    @Column(name = "LOWEST_SAL")
    private Integer lowestSal;
    @Nullable
    @Column(name = "HIGHEST_SAL")
    private Integer highestSal;

    @Nullable
    public String getGradeLevel() {
        return gradeLevel;
    }

    public void setGradeLevel(@Nullable String gradeLevel) {
        this.gradeLevel = gradeLevel;
    }

    @Nullable
    public Integer getLowestSal() {
        return lowestSal;
    }

    public void setLowestSal(@Nullable Integer lowestSal) {
        this.lowestSal = lowestSal;
    }

    @Nullable
    public Integer getHighestSal() {
        return highestSal;
    }

    public void setHighestSal(@Nullable Integer highestSal) {
        this.highestSal = highestSal;
    }
}

File: frontend/src/app/components/job-grades/job-grade.ts

export interface JobGradeInterface {
  gradeLevel: string;
  lowestSal: number;
  highestSal: number;
}

export class JobGrade {
  constructor(fields: Partial<JobGradeInterface>) {
    this.gradeLevel = fields.gradeLevel;
    this.lowestSal = fields.lowestSal;
    this.highestSal = fields.highestSal
  }
  gradeLevel: string;
  lowestSal: number;
  highestSal: number;

}
